src/js/components/project-view/empty-project.tsx

```diff
@@ -15,9 +16,72 @@ const EmptyProject = ({ project }: Props) => {
   return (
     <section className="container">
       <SectionTitle><span>Get started</span></SectionTitle>
-      <div className="row center-xs">
+      <div className="row start-xs">
         <div className="col-xs-12 col-md-8 col-lg-6">
-          <SetupInstructions project={project} styles={styles} hideLabels={true} />
+              <div className={styles.instructions}>
+            <div className={styles.section}>
+              <div className={styles.text}>
+                The URL for this project's Git repository is
+                <div className={styles.url}>{project.repoUrl}</div>
+              </div>
+              <div className={styles.heading}>
+                <Icon className={styles['heading-icon']} name="file-o" />
+                I'm starting a new project
+              </div>
+              <div className={styles.text}>
+                Connect your project folder with
+              </div>
+              <div className={styles.code}>
```

Comment:
Should the `.code` class be set to `width: 100%`? The current UI where the code blocks are different widths depending on the contents looks a bit off.